coral-schema/src/main/java/com/linkedin/coral/schema/avro/RelToAvroSchemaConverter.java

```diff
@@ -154,14 +155,16 @@ public Schema convert(@Nonnull RelNode relNode, boolean strictMode) {
   private static class SchemaRelShuttle extends RelShuttleImpl {
     private final Map<RelNode, Schema> schemaMap;
     private final boolean strictMode;
+    private final boolean forceLowercase;
 
     private final HiveMetastoreClient hiveMetastoreClient;
 
-    public SchemaRelShuttle(HiveMetastoreClient hiveMetastoreClient, Map<RelNode, Schema> schemaMap,
-        boolean strictMode) {
+    public SchemaRelShuttle(HiveMetastoreClient hiveMetastoreClient, Map<RelNode, Schema> schemaMap, boolean strictMode,
```

Comment:
Same as above should consider adding a new signature.

Reply:
This contains implementation details and is inside a private class, it is not likely be called outside.